stateEstimation/NavStateDot.m

```diff
@@ -49,9 +49,9 @@
 x_dot(4:6) = euler_to_bodyRates(X(4:6), -1) * (wb  - R_t2b*earth_rate_t);
 
 % dot{v_{b/t}^{b}} %
-x_dot(7:9) = IMU_to_body*u + G_b;
-             %- R_t2b*cross(earth_rate_t, R_t2b'*X(7:9)) ...
-             %- cross(wb, X(7:9));
+x_dot(7:9) = IMU_to_body*u + G_b ...
+             - R_t2b*cross(earth_rate_t, R_t2b'*X(7:9)) ...
```

Comment:
This term is correct here, but not in the navigation paper we're writing please update this in paper too !